composer.json

```diff
@@ -44,7 +45,20 @@
 		},
 		"bamarni-bin": {
 			"target-directory": "bin/composer"
-		}
+		},
+        "changelogger": {
+        	"formatter": "WCAdmin",
+			"versioning": "wordpress",
+        	"changes-dir": "./changelogs",
+            "types": [
+                "Fix",
+                "Add",
+                "Update",
+                "Dev",
+                "Tweak",
+                "Performance"
```

Comment:
We should also add `Enhancement` to this list

Reply:
Updated in https://github.com/woocommerce/woocommerce-admin/pull/6870/commits/2feaa328a2d2b2ec23457d0b40a754a6ee37d043